service/src/geopm_hash.h

```diff
@@ -19,21 +21,34 @@ extern "C"
 /***************************/
 enum geopm_region_hash_e {
     GEOPM_REGION_HASH_INVALID  = 0x0ULL,
-    GEOPM_REGION_HASH_UNMARKED = 0x725e8066ULL, /* Note the value is the geopm_crc32_str() of the stringified enum */
+    GEOPM_REGION_HASH_UNMARKED = 0x725e8066ULL,
```

Comment:
```suggestion
    GEOPM_REGION_HASH_UNMARKED = 0x725e8066ULL, /* geopm::hash("GEOPM_REGION_HASH_UNMARKED"); */
```

Reply:
Also, the hashed value should be updated since it changes.  Then similar changes should be made below for GEOPM_REGION_HASH_EPOCH and GEOPM_REGION_HASH_APP.

If you'd rather not change these numbers, then please add a comment explaning that these hashes were generated from our old crc32 implementation by passing the stringified enum.

I'm just concerned we'll want to know where these numbers came from at some point in the future.

Reply:
Hmm...I was just hoping to leave them as random.  I don't see why we will need to know were they came from in the future...

Reply:
I've added a detailed note instead